plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -138,39 +135,16 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
     if (max === undefined) max = 1;
   }
 
-  let tooltipFormatter: TimeFormatter | StringConstructor;
-  if (tooltipTimeFormat === smartDateFormatter.id) {
-    tooltipFormatter = smartDateDetailedFormatter;
-  } else if (tooltipTimeFormat) {
-    tooltipFormatter = getTimeFormatter(xAxisTimeFormat);
-  } else {
-    tooltipFormatter = String;
-  }
-
-  let xAxisFormatter: TimeFormatter | StringConstructor | undefined;
-  if (xAxisTimeFormat === smartDateFormatter.id || !xAxisTimeFormat) {
-    xAxisFormatter = undefined;
-  } else if (xAxisTimeFormat) {
-    xAxisFormatter = getTimeFormatter(xAxisTimeFormat);
-  } else {
-    xAxisFormatter = String;
-  }
+  const tooltipFormatter = getTooltipFormatter(tooltipTimeFormat);
+  const xAxisFormatter = getXAxisFormatter(xAxisTimeFormat);
```

Comment:
These were moved into the `transformers.ts` util so they could be utilized by the new chart